extension/src/test/cli/util.ts

```diff
@@ -36,7 +36,7 @@ export const initializeEmptyRepo = async (): Promise<string> => {
     return ''
   }
 
-  await gitExecutor.init(TEMP_DIR)
+  await gitExecutor.gitInit(TEMP_DIR)
```

Comment:
[F] (unfortunate but) you can't register the same command name twice in `InternalCommands`